test/cpp_extensions/open_registration_extension/pytorch_openreg/_device_daemon.py

```diff
@@ -56,6 +56,10 @@ def tensor_from_meta(self, meta):
                         storage_offset=0,
                     )
 
+        # Might be an empty tensor
+        if found_base is None and meta.nelem_in_bytes == 0:
+            found_base = torch.tensor((), dtype=torch.uint8)
```

Comment:
This doesn't do what you think, the numel of this tensor is 1 as it's a scalar tensor

Reply:
I've checked the doc https://github.com/pytorch/pytorch/blob/main/torch/_torch_docs.py#L9066 and try it locally. It seems `torch.tensor(())` is an empty tensor and its numel is 0. Did I get that right?

Reply:
oh oops, you are right!